src/Catel.MVVM/Catel.MVVM.Shared/Services/Extensions/IPleaseWaitServiceExtensions.cs

```diff
@@ -54,11 +55,11 @@ public static IDisposable HideTemporarily(this IPleaseWaitService pleaseWaitServ
         /// }
         /// ]]>
         /// </example>
-        public static IDisposable PushInScope(this IPleaseWaitService pleaseWaitService)
+        public static IDisposable PushInScope(this IPleaseWaitService pleaseWaitService, string status = "")
```

Comment:
Note that this is a breaking change, but it's acceptable because this will be available in v5. If you want this to be available in v4.5.4, you will need to introduce a new method without a default value and create a new PR.